client/src/about/custom-elements.ts

```diff
@@ -59,13 +58,16 @@ export class TeamMember extends LitElement {
     super.connectedCallback();
     this.tabIndex = 0;
     this._setID();
-    this.addEventListener("focus", this._focus);
+    if (window.location.hash === `#${this.id}`) {
+      setTimeout(() => this.focus(), 100);
```

Comment:
I think we have something competing here, or we're not yet in the DOM. Take a look @LeoMcA 

If we keep it we might wanna manage it the _lit_ way.